src/Management/src/EndpointCore/Extensions/ServiceCollectionExtensions.netcore31.cs

```diff
@@ -1,34 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the Apache 2.0 License.
-// See the LICENSE file in the project root for more information.
-
-#if NETCOREAPP3_1
-using Steeltoe.Management;
-using Steeltoe.Management.Endpoint;
-using Steeltoe.Management.Endpoint.Internal;
-
-namespace Microsoft.Extensions.DependencyInjection;
-
-public static class ServiceCollectionExtensions
-{
-    /// <summary>
-    /// Registers mapping for an endpoint. This gets used when calling <see cref="ActuatorRouteBuilderExtensions.MapAllActuators(AspNetCore.Routing.IEndpointRouteBuilder, MediaTypeVersion)"/>.
-    /// </summary>
-    /// <typeparam name="TEndpoint">The type of endpoint.</typeparam>
-    /// <param name="services">The service collection.</param>
-    /// <returns>A reference to the service collection.</returns>
-    public static IServiceCollection AddActuatorEndpointMapping<TEndpoint>(this IServiceCollection services)
```

Comment:
Is it a bad idea to be _removing_ these thing that are .net core 3.1 only? Could/should we handle this situation differently?

Reply:
What do you have in mind that possibly makes removing them a bad idea?

Reply:
I'm just thinking about the case where somebody is updating Steeltoe and their target framework at or around the same time... maybe it's not an issue since 4.0 only targets .net 6 and they wouldn't be able to. Wondering if there's a way we could realistically [Obsolete] the code with a hint where to go next

Reply:
I wouldn't worry about that, this happens all the time. But it would be good to mention in the release notes, though. That's the first place I look after updating and finding that the code no longer compiles. If we list breaking changes in a document, then we don't need to keep compiling/testing/etc the old code.

To be honest, I expected a different concern: a user is updating to .NET 6, but not adopting the new `Program.cs` without `Startup.cs`. So still using the old host builder, which is a scenario still supported by ASP.NET. If we do we want to support that scenario, we need to keep the `IServiceCollection` extension methods, as well as the ones on host builder.

Reply:
I _think_ that's already handled, but I don't know offhand - @hananiel do you have a thought here?

Reply:
Well if the code was already in a `#if`, scoping it down to .NET Core, then we haven't supported that scenario until now, therefore I think we don't need to start supporting it now. But maybe this varies on a case-by-case basis. Sounds like something to take into account during public API review.

Reply:
Oh.. re-reading this and looking at the other file, I don't think this comment belonged here at all. This particular example is a total non-issue for any of the concerns we've discussed

Reply:
We should revisit this conversation in https://github.com/SteeltoeOSS/Steeltoe/issues/827